terraform/modules/elasticache/redis.tf

```diff
@@ -25,24 +22,3 @@ resource "aws_elasticache_replication_group" "this" {
   subnet_group_name           = var.redis_subnet_group_name
   transit_encryption_enabled  = true
 }
-
-resource "aws_elasticache_cluster" "this" {

```

Comment:
We plan to set `highly_available` for Redis to `false` on staging, but I noticed that meant we'd use the `aws_elasticache_cluster` resource for creating Redis instead of `aws_elasticache_replication_group`, and this would mean losing encryption at rest as it can't be configured on there. 

You can use the replication_group resource for non-high and high availability, so I've updated this here to ensure we retain the encryption. [See this issue](https://github.com/hashicorp/terraform-provider-aws/issues/22123) for more info.